readthedocs/templates/search/elastic_search.html

```diff
@@ -25,226 +25,227 @@
 
 {% block content %}
 
-{% trans "100" as MAX_SUBSTRING_LIMIT %}
-
-<div class="navigable">
-  <ul>
-    <h5>{% trans 'Object Type' %}</h5>
-    {% if not project_obj %}
-      <li class="{% if type == 'project' %}active{% endif %}"><a href="?{% url_replace request 'type' 'project' %}">{% trans 'Projects' %}</a></li>
-    {% endif %}
-      <li class="{% if type == 'file' %}active{% endif %}"><a href="?{% url_replace request 'type' 'file' %}">{% trans 'Files' %}</a></li>
-    <hr>
-
-    {% if facets.index %}
-      <h5>{% trans 'Index' %}</h5>
-      {% for name, count, selected in facets.index %}
-        <li class="{% if selected %}active{% endif %} index-list">
-          {% if index == name %}
-            <a href="?{% url_replace request 'index' ''  %}">{{ name }}
-          {% else %}
-            <a href="?{% url_replace request 'index' name  %}">{{ name }}
-          {% endif %}
-          ({{ count }})
-          </a>
-        </li>
-      {% endfor %}
+{% with "100" as MAX_SUBSTRING_LIMIT %}
```

Comment:
Why did this file change so much?

Reply:
Every line is indented one more level.

Reply:
Also I have corrected the indentation of the whole file.